src/cards/repo-card.js

```diff
@@ -134,10 +137,13 @@ const renderRepoCard = (repo, options = {}) => {
     gap: 25,
   }).join("");
 
+  // Calculate the card width and height based on the provided arguments or defaults
+  const width = card_width || 400;
```

Comment:
We also need to apply a minimum card width. Similar to is done in the langs and stats cards.